Packs/GenericSQL/Integrations/GenericSQL/GenericSQL_description.md

```diff
@@ -0,0 +1,22 @@
+## Generic SQL
+Support the databases - mysql, PostgreSQL, Microsoft SQL and Oracle.
```

Comment:
Tested on the databases - mysql, PostgreSQL, Microsoft SQL and Oracle.
Compatible to sql alchemy supported dialects.

@kirbles19 your help is needed once again.